docs/api/input.md

```diff
@@ -24,341 +17,110 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
-            <IonInput value={text} placeholder="Enter Input" onIonChange={e => setText(e.detail.value!)}></IonInput>
-          </IonItem>
-
-          <IonItemDivider>Input with clear button when there is a value</IonItemDivider>
-          <IonItem>
-            <IonInput value={text} placeholder="Enter Input" onIonChange={e => setText(e.detail.value!)} clearInput></IonInput>
-          </IonItem>
-
-          <IonItemDivider>Number type input</IonItemDivider>
-          <IonItem>
-            <IonInput type="number" value={number} placeholder="Enter Number" onIonChange={e => setNumber(parseInt(e.detail.value!, 10))}></IonInput>
-          </IonItem>
-
-          <IonItemDivider>Disabled input</IonItemDivider>
-          <IonItem>
-            <IonInput value={text} disabled></IonInput>
-          </IonItem>
-
-          <IonItemDivider>Readonly input</IonItemDivider>
-          <IonItem>
-            <IonInput value={text} readonly></IonInput>
-          </IonItem>
-
-          <IonItemDivider>Inputs with labels</IonItemDivider>
-
-          <IonItem>
-            <IonLabel>Default Label</IonLabel>
-            <IonInput></IonInput>
-          </IonItem>
-
-          <IonItem>
-            <IonLabel position="floating">Floating Label</IonLabel>
-            <IonInput value={text}></IonInput>
-          </IonItem>
-
-          <IonItem>
-            <IonLabel position="fixed">Fixed Label</IonLabel>
-            <IonInput value={text}></IonInput>
-          </IonItem>
-
-          <IonItem>
-            <IonLabel position="stacked">Stacked Label</IonLabel>
-            <IonInput value={text}> </IonInput>
-          </IonItem>
-        </IonList>
-      </IonContent>
-    </IonPage>
-  );
-};
-```
+<!-- Reuse the playground from the Item directory -->
+import Counter from '@site/static/usage/item/counter/index.md';
 
-</TabItem>
+<Counter />
 
 
-<TabItem value="stencil">
+## Theming
 
-```tsx
-import { Component, h } from '@stencil/core';
+### Colors
 
-@Component({
-  tag: 'input-example',
-  styleUrl: 'input-example.css'
-})
-export class InputExample {
-  render() {
-    return [
-      // Default Input
-      <ion-input></ion-input>,
+import Colors from '@site/static/usage/input/theming/colors/index.md';
 
-      // Input with value
-      <ion-input value="custom"></ion-input>,
+<Colors />
 
-      // Input with placeholder
-      <ion-input placeholder="Enter Input"></ion-input>,
+### CSS Custom Properties
 
-      // Input with clear button when there is a value
-      <ion-input clearInput value="clear me"></ion-input>,
+Input uses scoped encapsulation, which means it will automatically scope its CSS by appending each of the styles with an additional class at runtime. Overriding scoped selectors in CSS requires a [higher specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) selector. Targeting the `ion-input` for customization will not work, therefore we recommend adding a class and customizing it that way.
```

Comment:
```suggestion
Input uses scoped encapsulation, which means it will automatically scope its CSS by appending each of the styles with an additional class at runtime. Overriding scoped selectors in CSS requires a [higher specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) selector. Targeting the `ion-input` for customization will not work; therefore we recommend adding a class and customizing it that way.
```